File: tests/fixtures/lang_samples/typescript/conditionals.ts
export function threeOps(a: boolean, b: boolean, c: boolean, d: boolean): number {
  if (a && b && c && d) {
    return 1;
  }
  return 0;
}

export function sixOps(
  a: boolean,
  b: boolean,
  c: boolean,
  d: boolean,
  e: boolean,
  f: boolean,
  g: boolean,
): number {
  while (a && b && c && d && e && f && g) {
    return 1;
  }
  return 0;
}

export function twoOps(a: boolean, b: boolean, c: boolean): number {
  if (a && b || c) {
    return 1;
  }
  return 0;
}

File: tests/fixtures/lang_samples/rust/conditionals.rs
pub fn three_ops(a: bool, b: bool, c: bool, d: bool) -> i32 {
    if a && b && c && d {
        return 1;
    }
    0
}

pub fn six_ops(a: bool, b: bool, c: bool, d: bool, e: bool, f: bool, g: bool) -> i32 {
    while a && b && c && d && e && f && g {
        return 1;
    }
    0
}

pub fn two_ops(a: bool, b: bool, c: bool) -> i32 {
    if a && b || c {
        return 1;
    }
    0
}
